data/mods/More_Survival_Tools/items.json

```diff
@@ -99,7 +99,20 @@
     "turns_per_charge": 600,
     "ammo": "tinder",
     "revert_to": "tinderbox",
-    "use_action": "TINDERBOX_LIT"
```

Comment:
If you're going to remove the reference to this function, you should remove the entire function from the C++ files in /src/.